### ISSUE TITLE
Refactor packages in plugin-merge

### ISSUE THREAD

**Issue by jeremykong-ur** [somahota/Speech-Based-Code-Editor-Project#8]
Currently:
_edu.brown.cs.voice2text_ contains all the Voice2Text code
_voice2code.parts_contains all the plugin related code

This is because xxx.parts is the default eclipse plugin package.
We should make this consistent. Sth like:
edu.brown.cs.voice2text
edu.brown.cs.text2code
edu.brown.cs.voice2codeplug

@Rashi1997 I think this is the convention for Java, but I am not sure...

@all feel free to add suggestions!

**Comment by Rashi1997** [comment]
Good point, was also thinking of separating utilities and config inside voice2text from main
